hugegraph-server/hugegraph-hbase/src/main/java/org/apache/hugegraph/backend/store/hbase/HbaseSessions.java

```diff
@@ -391,7 +391,10 @@ default R scan(String table, long limit) {
          * Scan records by rowkey prefix from a table
          */
         default R scan(String table, byte[] prefix) {
-            return this.scan(table, prefix, true, prefix);
+            // TODO: setRowPrefixFilter deprecated since HBase 2.5.0,
+            // will be removed in 4.0.0,setStartStopRowForPrefixScan(byte[]) instead.
```

Comment:
expect style like this:
```
/*
 * TODO: setRowPrefixFilter() deprecated since HBase 2.5.0, will be removed in 4.0.0,
 * use setStartStopRowForPrefixScan(byte[]) instead.
 * /
```